src/adapter/z-stack/znp/znp.ts

```diff
@@ -43,6 +43,7 @@ const autoDetectDefinitions = [
     {manufacturer: 'Texas Instruments', vendorId: '0451', productId: '16a8'}, // CC2531
     {manufacturer: 'Texas Instruments', vendorId: '0451', productId: 'bef3'}, // CC1352P_2 and CC26X2R1
     {manufacturer: 'Electrolama', vendorId: '0403', productId: '6015'}, // ZZH
+    { manufacturer: 'Silicon Labs', vendorId: '10c4', productId: 'ea60' }, // CC2652P
```

Comment:
I don't think we can add this one, `ea60` is from the CP210x which is also used in EmberZNet adapters.

Reply:
Mhh, it is however added there with a different manufacturer: https://github.com/Koenkk/zigbee-herdsman/blob/e2b4f9ca891bb1f1dd86c2b4e0570af5ec28707a/src/adapter/ember/adapter/emberAdapter.ts#L246

Reply:
I see, the Dongle-P also has a different ID so let's merge this, can you fix the lint issues?

Reply:
> can you fix the lint issues?

Done!

Reply:
Note: Can't use manufacturer as a "reference" for auto-detect, OS drivers can overwrite it. Which is why auto detect often fails btw.